specification/cognitiveservices/data-plane/TranslatorText/Batch/preview/v1.0/TranslatorBatch.json

```diff
@@ -0,0 +1,1469 @@
+          "type": "string",
+          "example": "https://myblob.blob.core.windows.net/Container/"
+        },
+        "filter": {
+          "$ref": "#/definitions/DocumentFilter"
+        },
+        "language": {
+          "description": "Language code\r\nIf none is specified, we will perform auto detect on the document",
+          "type": "string",
+          "example": "en"
+        },
+        "storageSource": {
+          "$ref": "#/definitions/StorageSource"
+        }
+      }
+    },
+    "Glossary": {
+      "description": "Glossary / translation memory for the request",
+      "required": [
+        "glossaryUrl"
+      ],
+      "type": "object",
+      "properties": {
+        "glossaryUrl": {
+          "description": "Location of the glossary. \r\nWe will use the file extension to extract the formating if the format parameter is not supplied.\r\n\r\nIf the translation language pair is not present in the glossary, it will not be applied",
+          "type": "string",
+          "example": "https://myblob.blob.core.windows.net/Container/myglossary.tsv"
+        },
+        "format": {
+          "description": "Format",
+          "type": "string",
+          "example": "XLIFF"
+        },
+        "version": {
+          "description": "Version",
+          "type": "string",
+          "example": "2.0"
+        }
+      }
+    },
+    "TargetInput": {
+      "description": "Destination for the finished translated documents",
+      "required": [
+        "language",
+        "targetUrl"
+      ],
+      "type": "object",
+      "properties": {
+        "targetUrl": {
+          "description": "Location of the folder / container with your documents",
+          "type": "string",
+          "example": "https://myblob.blob.core.windows.net/TargetUrl/"
+        },
+        "category": {
+          "description": "Category / custom system for translation request",
+          "type": "string",
+          "example": "general"
+        },
+        "language": {
+          "description": "Target Language",
+          "type": "string",
+          "example": "fr"
+        },
+        "glossaries": {
+          "description": "List of Glossary",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Glossary"
+          }
+        },
+        "storageSource": {
+          "$ref": "#/definitions/StorageSource"
+        }
+      }
+    },
+    "BatchRequest": {
+      "description": "Definition for the input batch translation request",
+      "required": [
+        "source",
+        "targets"
+      ],
+      "type": "object",
+      "properties": {
+        "source": {
+          "$ref": "#/definitions/SourceInput"
+        },
+        "targets": {
+          "description": "Location of the destination for the output",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TargetInput"
+          }
+        }
+      }
+    },
+    "JobSubmissionBatchRequest": {
```

Comment:
Now that `job` isn't in an endpoint name, it would make sense to rename this type.

Reply:
BatchSubmissionRequest